Pages/Grids/grids.html

```diff
@@ -0,0 +1,119 @@
+</head>
+
+<body>
+    <header>
+        <div>
+            <img class="ui-logo" src="/Assets/clean-logos.jpeg" />
+        </div>
+        <div class="right-icons">
+            <a href="/index.html" class="link-text-primary">Home</a>
+            <a href="/Components/docs.html" class="link-text-primary">Docs</a>
+            <div class="btn-toggle">
+                <i class="fas fa-sun"></i>
+            </div>
+        </div>
+    </header>
+
+    <!-- Aside -->
+    <aside>
+        <!-- Getting Started -->
+        <h3>Getting Started</h3>
+        <a href="/Components/docs.html">Installation</a>
+        <a href="/Pages/Text-Utilities/text.html">Typography</a>
+        <a class="active-tab" href="/Pages/Avatar/avatar.html">Grids</a>
+
+        <!-- Components -->
+        <h3>Components</h3>
+        <a href="/Pages/Avatar/avatar.html">Avatar</a>
+        <a href="/Pages/Alert/alert.html">Alert</a>
+        <a href="/Pages/Button/button.html">Button</a>
+        <a href="/Pages/Badge/badge.html">Badge</a>
+        <a href="/Pages/Card/card.html">Card</a>
+        <a href="/Pages/Input/input.html">Input</a>
+        <a href="/Pages/Image/image.html">Image</a>
+        <a href="/Pages/List/list.html">Lists</a>
+        <a href="/Pages/Modal/modal.html">Modal</a>
+        <a href="/Pages/Toast/toast.html">Toast</a>
+        <a href="/Pages/Rating/rating.html">Rating</a>
+    </aside>
+
+    <!-- Main -->
+    <main>
+        <h1>Grids</h1>
+        <p class="body-lg">Grids can be used for creating layouts easily.</p>
+
+        <h2 class="section-header">50-50 Grid</h2>
+        <section class="section-grid">
+            <div class="grid-50-50">
+                <div>
+                    <img class="grid-img" src="/Assets/grid//image50.jpg" alt="responsive-img" />
+                </div>
+                <div class="div-txt">
+                    This is an example of 50-50 Grid. Here we have an Image and some Text !!
+                </div>
+            </div>
+        </section>
+
+        <p class="body-r">Copy the given code to create a 50-50 Grid</p>
+        <iframe
+  src="https://carbon.now.sh/embed?bg=rgba%283%2C26%2C77%2C1%29&t=blackboard&wt=none&l=htmlmixed&width=680&ds=false&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Fira+Code&fs=13px&lh=152%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522grid-50-50%2522%253E%250A%2509%253Cdiv%253E%250A%2509%2509%253Cimg%2520src%253D%2522https%253A%252F%252Fpicsum.photos%252F500%252F500%2522%2520alt%253D%2522responsive-img%2522%2520%252F%253E%250A%2509%253C%252Fdiv%253E%250A%2520%2520%2520%2520%253Cdiv%2520class%253D%2522div-txt%2522%253E%250A%2520%2520%2520%2520%2509This%2520is%2520an%2520example%2520of%252050-50%2520Grid.%2520Here%2520we%2520have%2520an%2520Image%2520and%2520some%2520Text%21%21%250A%2509%253C%252Fdiv%253E%250A%253C%252Fdiv%253E"
+  style="width: 80rem; height: 40rem; border:0; transform: scale(1); overflow:hidden;"
```

Comment:
suggestion : try to avoid inline css

Reply:
> suggestion : try to avoid inline css

I didn't do this styling, it's imported from carbon